packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -198,10 +212,11 @@ const DesignButton: React.FC< DesignButtonProps > = ( {
 		if ( designIsBundledWithWoo ) {
 			badge = <WooCommerceBundledBadge />;
 		} else if ( isPremium ) {
-			const toolTip =
-				shouldUpgrade && preselected_style_variation
-					? __( 'Unlock this style, and tons of other features, by upgrading to a Premium plan.' )
-					: undefined;
+			const showTooltip =
```

Comment:
Nitpick: This is perhaps a tiny bit misleading - a tooltip might still be shown even if showTooltip is false. It governs whether to use the custom style variation tooltip rather than whether to use a tooltip at all.

Reply:
Good point – changed in https://github.com/Automattic/wp-calypso/pull/73476/commits/5165a6fa5fac9a9783331417a0354e6117c039ec